drivers/usb/device/usb_dc_stm32.c

```diff
@@ -309,6 +317,20 @@ int usb_dc_attach(void)
 		return ret;
 	}
 
+	/* Enable VDDUSB if available */
```

Comment:
We should mention in the comment that we need the following code for stm32l4 series and why we need it (a part from DM00310109 section 5.1.3 should do it).